  INFO root [2019-09-19 08:54:31,864] - 
  INFO root [2019-09-19 08:54:31,865] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 08:54:32,688] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 08:54:32,688] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 08:54:32,688] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 08:54:33,381] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54320 )
  INFO root [2019-09-19 08:54:33,382] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54320, SOMAXCONN=5.
  INFO root [2019-09-19 08:54:33,383] - server-configs:
  INFO root [2019-09-19 08:54:33,383] - 	internalTcpAddr : 192.168.43.114:54320
  INFO root [2019-09-19 08:54:33,383] - 	componentID : 5000
  WARN root [2019-09-19 08:54:33,384] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 08:54:35,031] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 8104 ----
  INFO root [2019-09-19 08:54:35,032] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 08:54:35,075] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 08:54:35,075] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 08:54:35,804] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 08:54:35,804] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 08:54:42,317] - TCPPacketReceiver::processPendingEvents(192.168.43.114:54331): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 08:54:43,316] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 08:54:43,409] - Components::findComponents: found logger, addr:192.168.43.114:54323
  INFO root [2019-09-19 08:54:43,410] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 08:54:43,410] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 08:54:43,411] - Components::findComponents: find dbmgr(2)...
  WARN root [2019-09-19 08:54:44,518] - Forwarding logs to logger[192.168.43.114:54323/0/0/0]...
 ERROR root [2019-09-19 08:54:53,035] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459305733056, lastReceivedTime=459209488511, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 08:54:53,037] - The following logs sent to logger failed:
  WARN root [2019-09-19 08:54:53,037] - ==> [2019-09-19 08:54:53 034] TCPPacketReceiver::processPendingEvents(192.168.43.114:54323): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 08:54:53,037] - ==> [2019-09-19 08:54:53 036] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=459305733056, lastReceivedTime=459209488511, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 08:56:25,949] - 
  INFO root [2019-09-19 08:56:25,957] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 08:56:25,958] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 08:56:25,958] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 08:56:25,958] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 08:56:25,973] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54367 )
  INFO root [2019-09-19 08:56:25,973] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54367, SOMAXCONN=5.
  INFO root [2019-09-19 08:56:25,973] - server-configs:
  INFO root [2019-09-19 08:56:25,973] - 	internalTcpAddr : 192.168.43.114:54367
  INFO root [2019-09-19 08:56:25,973] - 	componentID : 5000
  WARN root [2019-09-19 08:56:25,974] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 08:56:26,313] - Components::findLogger: found logger, addr:192.168.43.114:54363
  INFO root [2019-09-19 08:56:26,313] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 08:56:26,313] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 08:56:26,318] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 14804 ----
  INFO root [2019-09-19 08:56:26,319] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 08:56:26,319] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 08:56:26,319] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 08:56:26,760] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 08:56:26,761] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 08:56:32,897] - Forwarding logs to logger[192.168.43.114:54363/0/0/0]...
 ERROR root [2019-09-19 08:56:48,093] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=460456310263, lastReceivedTime=460238520283, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 08:56:48,095] - The following logs sent to logger failed:
  WARN root [2019-09-19 08:56:48,095] - ==> [2019-09-19 08:56:48 091] TCPPacketReceiver::processPendingEvents(192.168.43.114:54363): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 08:56:48,096] - ==> [2019-09-19 08:56:48 094] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=460456310263, lastReceivedTime=460238520283, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 08:56:48,096] - Components::findComponents: find dbmgr(8)...
  INFO root [2019-09-19 08:59:58,808] - 
  INFO root [2019-09-19 08:59:58,810] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 08:59:58,811] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 08:59:58,811] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 08:59:58,811] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 08:59:58,824] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54414 )
  INFO root [2019-09-19 08:59:58,824] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54414, SOMAXCONN=5.
  INFO root [2019-09-19 08:59:58,824] - server-configs:
  INFO root [2019-09-19 08:59:58,824] - 	internalTcpAddr : 192.168.43.114:54414
  INFO root [2019-09-19 08:59:58,824] - 	componentID : 5000
  WARN root [2019-09-19 08:59:58,824] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 09:00:00,329] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 12832 ----
  INFO root [2019-09-19 09:00:00,330] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:00:00,330] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:00:00,331] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:00:00,837] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 09:00:00,838] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:00:08,014] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:00:08,048] - Components::findComponents: found logger, addr:192.168.43.114:54412
  INFO root [2019-09-19 09:00:08,049] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:00:08,049] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:00:12,754] - 
  INFO root [2019-09-19 09:00:12,760] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 09:00:12,760] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 09:00:12,760] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 09:00:12,760] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 09:00:12,787] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54433 )
  INFO root [2019-09-19 09:00:12,788] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54433, SOMAXCONN=5.
  INFO root [2019-09-19 09:00:12,788] - server-configs:
  INFO root [2019-09-19 09:00:12,788] - 	internalTcpAddr : 192.168.43.114:54433
  INFO root [2019-09-19 09:00:12,788] - 	componentID : 5000
  WARN root [2019-09-19 09:00:12,792] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 09:00:13,152] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20191, errorno=10048
  INFO root [2019-09-19 09:00:13,156] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 10344 ----
  INFO root [2019-09-19 09:00:13,156] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:00:13,157] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:00:13,157] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:00:13,565] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 09:00:13,566] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:00:20,698] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:00:20,763] - Components::findComponents: found logger, addr:192.168.43.114:54435
  INFO root [2019-09-19 09:00:20,763] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:00:20,763] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:00:20,765] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 09:00:20,766] - Components::findComponents: found logger, addr:192.168.43.114:54435
  WARN root [2019-09-19 09:00:21,883] - Forwarding logs to logger[192.168.43.114:54435/0/0/0]...
  INFO root [2019-09-19 09:01:04,534] - 
  INFO root [2019-09-19 09:01:04,536] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 09:01:04,537] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 09:01:04,537] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 09:01:04,538] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 09:01:04,550] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:54448 )
  INFO root [2019-09-19 09:01:04,550] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:54448, SOMAXCONN=5.
  INFO root [2019-09-19 09:01:04,551] - server-configs:
  INFO root [2019-09-19 09:01:04,551] - 	internalTcpAddr : 192.168.43.114:54448
  INFO root [2019-09-19 09:01:04,551] - 	componentID : 5000
  WARN root [2019-09-19 09:01:04,552] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 09:01:06,058] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 17012 ----
  INFO root [2019-09-19 09:01:06,059] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:01:06,059] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:01:06,059] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:01:06,469] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 09:01:06,469] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:01:13,622] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:01:13,636] - Components::findComponents: found logger, addr:192.168.43.114:54452
  INFO root [2019-09-19 09:01:13,636] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:01:13,636] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:01:13,637] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 09:01:13,739] - Components::findComponents: found dbmgr, addr:192.168.43.114:54446
  INFO root [2019-09-19 09:01:13,740] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:01:13,740] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:01:13,741] - Components::findComponents: found dbmgr, addr:192.168.43.114:54446
  WARN root [2019-09-19 09:01:14,742] - Forwarding logs to logger[192.168.43.114:54452/0/0/0]...
 ERROR root [2019-09-19 09:22:57,374] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476149129437, lastReceivedTime=475921480232, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-19 09:22:57,378] - The following logs sent to logger failed:
  WARN root [2019-09-19 09:22:57,379] - ==> [2019-09-19 09:22:57 371] TCPPacketReceiver::processPendingEvents(192.168.43.114:54452): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:22:57,379] - ==> [2019-09-19 09:22:57 375] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476149129437, lastReceivedTime=475921480232, inactivityExceptionPeriod=599499617)
  WARN root [2019-09-19 09:22:57,449] - TCPPacketReceiver::processPendingEvents(192.168.43.114:54446): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:22:57,450] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=476149881106, lastReceivedTime=475920324122, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-19 09:22:57,457] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 09:22:57,459] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 09:22:57,459] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 09:22:57,486] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 09:23:58,964] - 
  INFO root [2019-09-19 09:23:58,966] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 09:23:58,968] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 09:23:58,968] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 09:23:58,968] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 09:23:58,992] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.43.114 ( 192.168.43.114:55124 )
  INFO root [2019-09-19 09:23:58,992] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.43.114:55124, SOMAXCONN=5.
  INFO root [2019-09-19 09:23:58,993] - server-configs:
  INFO root [2019-09-19 09:23:58,993] - 	internalTcpAddr : 192.168.43.114:55124
  INFO root [2019-09-19 09:23:58,993] - 	componentID : 5000
  WARN root [2019-09-19 09:23:58,993] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 09:24:00,500] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 11684 ----
  INFO root [2019-09-19 09:24:00,501] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 09:24:00,501] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 09:24:00,502] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 09:24:00,968] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 09:24:00,970] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 09:24:08,123] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 09:24:08,203] - Components::findComponents: found logger, addr:192.168.43.114:55122
  INFO root [2019-09-19 09:24:08,204] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:24:08,204] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:24:08,205] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 09:24:08,308] - Components::findComponents: found dbmgr, addr:192.168.43.114:55123
  INFO root [2019-09-19 09:24:08,308] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 09:24:08,308] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 09:24:08,308] - Components::findComponents: found dbmgr, addr:192.168.43.114:55123
  WARN root [2019-09-19 09:24:09,309] - Forwarding logs to logger[192.168.43.114:55122/0/0/0]...
 ERROR root [2019-09-19 09:33:13,985] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=482315229422, lastReceivedTime=482165022778, inactivityExceptionPeriod=599500192)
  INFO root [2019-09-19 09:33:13,990] - The following logs sent to logger failed:
  WARN root [2019-09-19 09:33:13,990] - ==> [2019-09-19 09:33:13 973] TCPPacketReceiver::processPendingEvents(192.168.43.114:55122): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:33:13,990] - ==> [2019-09-19 09:33:13 986] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=482315229422, lastReceivedTime=482165022778, inactivityExceptionPeriod=599500192)
  WARN root [2019-09-19 09:33:14,102] - TCPPacketReceiver::processPendingEvents(192.168.43.114:55123): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 09:33:14,104] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=482316428256, lastReceivedTime=482165032644, inactivityExceptionPeriod=599500192)
  INFO root [2019-09-19 09:33:14,131] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 09:33:14,134] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 09:33:14,134] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 09:33:14,135] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:05:50,152] - 
  INFO root [2019-09-19 19:05:50,701] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:05:51,371] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:05:51,371] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:05:51,371] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:05:51,472] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60617 )
  INFO root [2019-09-19 19:05:51,472] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60617, SOMAXCONN=5.
  INFO root [2019-09-19 19:05:51,472] - server-configs:
  INFO root [2019-09-19 19:05:51,472] - 	internalTcpAddr : 192.168.202.54:60617
  INFO root [2019-09-19 19:05:51,472] - 	componentID : 5000
  WARN root [2019-09-19 19:05:51,472] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:05:53,081] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 18384 ----
  INFO root [2019-09-19 19:05:53,081] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:05:53,182] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:05:53,182] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:05:54,232] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:05:54,241] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:06:01,532] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:06:01,552] - Components::findComponents: found logger, addr:192.168.202.54:60621
  INFO root [2019-09-19 19:06:01,557] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:06:01,557] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:06:01,559] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:06:01,672] - Components::findComponents: found dbmgr, addr:192.168.202.54:60625
  INFO root [2019-09-19 19:06:01,672] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:06:01,672] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:06:01,672] - Components::findComponents: found dbmgr, addr:192.168.202.54:60625
  WARN root [2019-09-19 19:06:02,692] - Forwarding logs to logger[192.168.202.54:60621/0/0/0]...
 ERROR root [2019-09-19 19:06:32,541] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=826394341851, lastReceivedTime=826298859942, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 19:06:32,552] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:06:32,602] - ==> [2019-09-19 19:06:32 550] TCPPacketReceiver::processPendingEvents(192.168.202.54:60621): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:06:32,602] - ==> [2019-09-19 19:06:32 552] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=826394341851, lastReceivedTime=826298859942, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 19:06:32,621] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60625): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:06:32,632] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=826395187088, lastReceivedTime=826307455655, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 19:06:32,657] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:06:32,657] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:06:32,657] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:06:32,661] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:10:14,850] - 
  INFO root [2019-09-19 19:10:14,856] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:10:14,857] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:10:14,857] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:10:14,858] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:10:14,883] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60734 )
  INFO root [2019-09-19 19:10:14,884] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60734, SOMAXCONN=5.
  INFO root [2019-09-19 19:10:14,884] - server-configs:
  INFO root [2019-09-19 19:10:14,884] - 	internalTcpAddr : 192.168.202.54:60734
  INFO root [2019-09-19 19:10:14,884] - 	componentID : 5000
  WARN root [2019-09-19 19:10:14,884] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:10:16,392] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 13644 ----
  INFO root [2019-09-19 19:10:16,393] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:10:16,393] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:10:16,394] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:10:16,828] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:10:16,828] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:10:23,956] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:10:23,979] - Components::findComponents: found logger, addr:192.168.202.54:60735
  INFO root [2019-09-19 19:10:23,979] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:10:23,979] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:10:23,980] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:10:24,083] - Components::findComponents: found dbmgr, addr:192.168.202.54:60737
  INFO root [2019-09-19 19:10:24,083] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:10:24,084] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:10:24,084] - Components::findComponents: found dbmgr, addr:192.168.202.54:60737
  WARN root [2019-09-19 19:10:25,084] - Forwarding logs to logger[192.168.202.54:60735/0/0/0]...
 ERROR root [2019-09-19 19:12:06,430] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=829733136542, lastReceivedTime=829517752260, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:12:06,435] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:12:06,436] - ==> [2019-09-19 19:12:06 428] TCPPacketReceiver::processPendingEvents(192.168.202.54:60735): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:12:06,436] - ==> [2019-09-19 19:12:06 431] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=829733136542, lastReceivedTime=829517752260, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:12:06,561] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60737): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:12:06,563] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=829734459882, lastReceivedTime=829518410920, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:12:06,565] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:12:06,595] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:12:06,596] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:12:06,597] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:14:11,431] - 
  INFO root [2019-09-19 19:14:11,433] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:14:11,434] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:14:11,435] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:14:11,435] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:14:11,444] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60801 )
  INFO root [2019-09-19 19:14:11,444] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60801, SOMAXCONN=5.
  INFO root [2019-09-19 19:14:11,444] - server-configs:
  INFO root [2019-09-19 19:14:11,445] - 	internalTcpAddr : 192.168.202.54:60801
  INFO root [2019-09-19 19:14:11,445] - 	componentID : 5000
  WARN root [2019-09-19 19:14:11,445] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:14:12,951] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 11896 ----
  INFO root [2019-09-19 19:14:12,951] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:14:12,952] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:14:12,952] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:14:13,391] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:14:13,392] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:14:20,513] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:14:20,529] - Components::findComponents: found logger, addr:192.168.202.54:60804
  INFO root [2019-09-19 19:14:20,529] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:14:20,529] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:14:20,530] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:14:20,635] - Components::findComponents: found dbmgr, addr:192.168.202.54:60800
  INFO root [2019-09-19 19:14:20,635] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:14:20,636] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:14:20,637] - Components::findComponents: found dbmgr, addr:192.168.202.54:60800
  WARN root [2019-09-19 19:14:21,638] - Forwarding logs to logger[192.168.202.54:60804/0/0/0]...
 ERROR root [2019-09-19 19:15:48,605] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=831954887422, lastReceivedTime=831883886195, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:15:48,608] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:15:48,608] - ==> [2019-09-19 19:15:48 603] TCPPacketReceiver::processPendingEvents(192.168.202.54:60804): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:15:48,608] - ==> [2019-09-19 19:15:48 606] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=831954887422, lastReceivedTime=831883886195, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:15:48,693] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60800): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:15:48,695] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=831955778011, lastReceivedTime=831882841461, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:15:48,709] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:15:48,712] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:15:48,713] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:15:48,714] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:16:41,319] - 
  INFO root [2019-09-19 19:16:41,321] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:16:41,322] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:16:41,322] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:16:41,322] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:16:41,333] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60888 )
  INFO root [2019-09-19 19:16:41,333] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60888, SOMAXCONN=5.
  INFO root [2019-09-19 19:16:41,334] - server-configs:
  INFO root [2019-09-19 19:16:41,334] - 	internalTcpAddr : 192.168.202.54:60888
  INFO root [2019-09-19 19:16:41,334] - 	componentID : 5000
  WARN root [2019-09-19 19:16:41,334] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:16:42,840] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 8156 ----
  INFO root [2019-09-19 19:16:42,840] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:16:42,840] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:16:42,841] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:16:43,283] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:16:43,284] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:16:49,412] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60907): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 19:16:50,422] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:16:50,439] - Components::findComponents: found logger, addr:192.168.202.54:60893
  INFO root [2019-09-19 19:16:50,440] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:16:50,440] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:16:50,441] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:16:50,544] - Components::findComponents: found dbmgr, addr:192.168.202.54:60886
  INFO root [2019-09-19 19:16:50,544] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:16:50,544] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:16:50,544] - Components::findComponents: found dbmgr, addr:192.168.202.54:60886
  WARN root [2019-09-19 19:16:51,544] - Forwarding logs to logger[192.168.202.54:60893/0/0/0]...
 ERROR root [2019-09-19 19:18:07,120] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=833340039640, lastReceivedTime=833082988335, inactivityExceptionPeriod=599497954)
  INFO root [2019-09-19 19:18:07,122] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:18:07,123] - ==> [2019-09-19 19:18:07 115] TCPPacketReceiver::processPendingEvents(192.168.202.54:60893): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:18:07,123] - ==> [2019-09-19 19:18:07 121] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=833340039640, lastReceivedTime=833082988335, inactivityExceptionPeriod=599497954)
  WARN root [2019-09-19 19:18:07,232] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60886): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:18:07,239] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=833341227545, lastReceivedTime=833082032984, inactivityExceptionPeriod=599497954)
  INFO root [2019-09-19 19:18:07,288] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:18:07,289] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:18:07,291] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:18:07,291] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:18:33,666] - 
  INFO root [2019-09-19 19:18:33,701] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:18:33,703] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:18:33,703] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:18:33,703] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:18:33,741] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:60961 )
  INFO root [2019-09-19 19:18:33,741] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:60961, SOMAXCONN=5.
  INFO root [2019-09-19 19:18:33,741] - server-configs:
  INFO root [2019-09-19 19:18:33,741] - 	internalTcpAddr : 192.168.202.54:60961
  INFO root [2019-09-19 19:18:33,741] - 	componentID : 5000
  WARN root [2019-09-19 19:18:33,751] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:18:34,357] - Components::findLogger: found logger, addr:192.168.202.54:60958
  INFO root [2019-09-19 19:18:34,358] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:18:34,358] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:18:34,369] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 1232 ----
  INFO root [2019-09-19 19:18:34,370] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:18:34,370] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:18:34,371] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:18:34,938] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:18:34,953] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:18:41,101] - Forwarding logs to logger[192.168.202.54:60958/0/0/0]...
 ERROR root [2019-09-19 19:21:41,840] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=835487236860, lastReceivedTime=835404281861, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 19:21:41,848] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:21:41,849] - ==> [2019-09-19 19:21:41 831] TCPPacketReceiver::processPendingEvents(192.168.202.54:60958): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:21:41,850] - ==> [2019-09-19 19:21:41 842] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=835487236860, lastReceivedTime=835404281861, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 19:21:42,054] - TCPPacketReceiver::processPendingEvents(192.168.202.54:60960): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:21:42,067] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=835489507250, lastReceivedTime=835405222305, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 19:21:42,078] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:21:42,083] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:21:42,085] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:21:42,086] - ThreadPool::destroy(): starting size 2.
  WARN root [2019-09-19 19:21:42,387] - ThreadPool::destroy(): waiting for thread(2)[0x142b708,0x142b9e0,], try=1
  INFO root [2019-09-19 19:26:03,846] - 
  INFO root [2019-09-19 19:26:03,857] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:26:03,860] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:26:03,860] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:26:03,860] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:26:03,897] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61046 )
  INFO root [2019-09-19 19:26:03,897] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61046, SOMAXCONN=5.
  INFO root [2019-09-19 19:26:03,897] - server-configs:
  INFO root [2019-09-19 19:26:03,898] - 	internalTcpAddr : 192.168.202.54:61046
  INFO root [2019-09-19 19:26:03,898] - 	componentID : 5000
  WARN root [2019-09-19 19:26:03,898] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:26:05,407] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 9420 ----
  INFO root [2019-09-19 19:26:05,407] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:26:05,408] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:26:05,409] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:26:06,080] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:26:06,081] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:26:13,269] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:26:13,294] - Components::findComponents: found logger, addr:192.168.202.54:61045
  INFO root [2019-09-19 19:26:13,294] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:26:13,295] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:26:13,299] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:26:13,404] - Components::findComponents: found dbmgr, addr:192.168.202.54:61047
  INFO root [2019-09-19 19:26:13,404] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:26:13,404] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:26:13,405] - Components::findComponents: found dbmgr, addr:192.168.202.54:61047
  WARN root [2019-09-19 19:26:14,405] - Forwarding logs to logger[192.168.202.54:61045/0/0/0]...
 ERROR root [2019-09-19 19:27:52,864] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=839197473665, lastReceivedTime=839005905899, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-19 19:27:52,876] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:27:52,876] - ==> [2019-09-19 19:27:52 861] TCPPacketReceiver::processPendingEvents(192.168.202.54:61045): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:27:52,877] - ==> [2019-09-19 19:27:52 865] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=839197473665, lastReceivedTime=839005905899, inactivityExceptionPeriod=599499489)
  WARN root [2019-09-19 19:27:52,967] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61047): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:27:52,977] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=839198609369, lastReceivedTime=839008022613, inactivityExceptionPeriod=599499489)
  INFO root [2019-09-19 19:27:53,009] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:27:53,010] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:27:53,010] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:27:53,011] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:29:35,831] - 
  INFO root [2019-09-19 19:29:35,839] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:29:35,841] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:29:35,841] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:29:35,841] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:29:35,880] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61127 )
  INFO root [2019-09-19 19:29:35,880] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61127, SOMAXCONN=5.
  INFO root [2019-09-19 19:29:35,880] - server-configs:
  INFO root [2019-09-19 19:29:35,880] - 	internalTcpAddr : 192.168.202.54:61127
  INFO root [2019-09-19 19:29:35,880] - 	componentID : 5000
  WARN root [2019-09-19 19:29:35,896] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:29:37,412] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 2116 ----
  INFO root [2019-09-19 19:29:37,412] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:29:37,413] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:29:37,413] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:29:38,019] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:29:38,046] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:29:44,202] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61144): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 19:29:45,204] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:29:45,308] - Components::findComponents: found logger, addr:192.168.202.54:61123
  INFO root [2019-09-19 19:29:45,308] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:29:45,308] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:29:45,312] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:29:45,316] - Components::findComponents: found logger, addr:192.168.202.54:61123
  INFO root [2019-09-19 19:29:45,321] - Components::findComponents: found dbmgr, addr:192.168.202.54:61125
  INFO root [2019-09-19 19:29:45,321] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:29:45,322] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:29:45,424] - Components::findComponents: found dbmgr, addr:192.168.202.54:61125
  WARN root [2019-09-19 19:29:46,425] - Forwarding logs to logger[192.168.202.54:61123/0/0/0]...
 ERROR root [2019-09-19 19:31:10,677] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=841175603250, lastReceivedTime=841126932562, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:31:10,679] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:31:10,680] - ==> [2019-09-19 19:31:10 676] TCPPacketReceiver::processPendingEvents(192.168.202.54:61123): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:31:10,680] - ==> [2019-09-19 19:31:10 677] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=841175603250, lastReceivedTime=841126932562, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:31:10,795] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61125): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:31:10,797] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=841176798258, lastReceivedTime=841127400221, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:31:10,812] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:31:10,824] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:31:10,825] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:31:10,825] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:34:51,179] - 
  INFO root [2019-09-19 19:34:51,190] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:34:51,191] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:34:51,191] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:34:51,191] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:34:51,294] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61221 )
  INFO root [2019-09-19 19:34:51,294] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61221, SOMAXCONN=5.
  INFO root [2019-09-19 19:34:51,295] - server-configs:
  INFO root [2019-09-19 19:34:51,295] - 	internalTcpAddr : 192.168.202.54:61221
  INFO root [2019-09-19 19:34:51,295] - 	componentID : 5000
  WARN root [2019-09-19 19:34:51,306] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:34:52,814] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 12736 ----
  INFO root [2019-09-19 19:34:52,815] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:34:52,815] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:34:52,816] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:34:53,388] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:34:53,398] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:35:00,561] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:35:00,644] - Components::findComponents: found logger, addr:192.168.202.54:61216
  INFO root [2019-09-19 19:35:00,645] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:35:00,645] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:35:00,647] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:35:00,658] - Components::findComponents: found dbmgr, addr:192.168.202.54:61214
  INFO root [2019-09-19 19:35:00,658] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:35:00,659] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:35:00,666] - Components::findComponents: found logger, addr:192.168.202.54:61216
  INFO root [2019-09-19 19:35:00,684] - Components::findComponents: found dbmgr, addr:192.168.202.54:61214
  WARN root [2019-09-19 19:35:01,685] - Forwarding logs to logger[192.168.202.54:61216/0/0/0]...
 ERROR root [2019-09-19 19:36:06,969] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=844138527140, lastReceivedTime=843979627712, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-19 19:36:06,975] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:36:06,976] - ==> [2019-09-19 19:36:06 964] TCPPacketReceiver::processPendingEvents(192.168.202.54:61216): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:36:06,976] - ==> [2019-09-19 19:36:06 970] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=844138527140, lastReceivedTime=843979627712, inactivityExceptionPeriod=599499617)
  WARN root [2019-09-19 19:36:07,070] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61214): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:36:07,080] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=844139640987, lastReceivedTime=843979322535, inactivityExceptionPeriod=599499617)
  INFO root [2019-09-19 19:36:07,111] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:36:07,116] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:36:07,116] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:36:07,117] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:38:55,531] - 
  INFO root [2019-09-19 19:38:55,533] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:38:55,535] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:38:55,535] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:38:55,535] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:38:55,579] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61303 )
  INFO root [2019-09-19 19:38:55,580] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61303, SOMAXCONN=5.
  INFO root [2019-09-19 19:38:55,580] - server-configs:
  INFO root [2019-09-19 19:38:55,580] - 	internalTcpAddr : 192.168.202.54:61303
  INFO root [2019-09-19 19:38:55,580] - 	componentID : 5000
  WARN root [2019-09-19 19:38:55,581] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:38:57,086] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 16768 ----
  INFO root [2019-09-19 19:38:57,087] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:38:57,087] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:38:57,088] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:38:57,504] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:38:57,507] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:38:57,862] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20553, errorno=10048
  INFO root [2019-09-19 19:39:04,989] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:39:05,012] - Components::findComponents: found logger, addr:192.168.202.54:61304
  INFO root [2019-09-19 19:39:05,012] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:39:05,012] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:39:05,014] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:39:05,017] - Components::findComponents: found logger, addr:192.168.202.54:61304
  INFO root [2019-09-19 19:39:05,020] - Components::findComponents: found dbmgr, addr:192.168.202.54:61299
  INFO root [2019-09-19 19:39:05,020] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:39:05,020] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:39:05,121] - Components::findComponents: found dbmgr, addr:192.168.202.54:61299
  WARN root [2019-09-19 19:39:06,121] - Forwarding logs to logger[192.168.202.54:61304/0/0/0]...
 ERROR root [2019-09-19 19:42:02,098] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=847689816558, lastReceivedTime=847624761247, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:42:02,118] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:42:02,118] - ==> [2019-09-19 19:42:02 095] TCPPacketReceiver::processPendingEvents(192.168.202.54:61304): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:42:02,118] - ==> [2019-09-19 19:42:02 099] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=847689816558, lastReceivedTime=847624761247, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:42:02,206] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61299): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:42:02,208] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=847690924812, lastReceivedTime=847624246039, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:42:02,227] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:42:02,239] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:42:02,240] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:42:02,240] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:43:42,129] - 
  INFO root [2019-09-19 19:43:42,130] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:43:42,131] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:43:42,131] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:43:42,131] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:43:42,149] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61376 )
  INFO root [2019-09-19 19:43:42,149] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61376, SOMAXCONN=5.
  INFO root [2019-09-19 19:43:42,149] - server-configs:
  INFO root [2019-09-19 19:43:42,150] - 	internalTcpAddr : 192.168.202.54:61376
  INFO root [2019-09-19 19:43:42,150] - 	componentID : 5000
  WARN root [2019-09-19 19:43:42,158] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:43:43,665] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 15600 ----
  INFO root [2019-09-19 19:43:43,666] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:43:43,666] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:43:43,667] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:43:44,127] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:43:44,135] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:43:50,256] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61392): Throwing REASON_GENERAL_NETWORK - WSAECONNABORTED
  INFO root [2019-09-19 19:43:51,269] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:43:51,283] - Components::findComponents: found logger, addr:192.168.202.54:61374
  INFO root [2019-09-19 19:43:51,283] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:43:51,283] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:43:51,284] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:43:51,387] - Components::findComponents: found dbmgr, addr:192.168.202.54:61372
  INFO root [2019-09-19 19:43:51,387] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:43:51,387] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:43:51,387] - Components::findComponents: found dbmgr, addr:192.168.202.54:61372
  WARN root [2019-09-19 19:43:52,388] - Forwarding logs to logger[192.168.202.54:61374/0/0/0]...
 ERROR root [2019-09-19 19:47:40,858] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=851077421735, lastReceivedTime=850790132860, inactivityExceptionPeriod=599497826)
  INFO root [2019-09-19 19:47:40,862] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:47:40,862] - ==> [2019-09-19 19:47:40 857] TCPPacketReceiver::processPendingEvents(192.168.202.54:61374): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:47:40,863] - ==> [2019-09-19 19:47:40 859] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=851077421735, lastReceivedTime=850790132860, inactivityExceptionPeriod=599497826)
  WARN root [2019-09-19 19:47:40,944] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61372): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:47:40,948] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=851078313980, lastReceivedTime=850789963626, inactivityExceptionPeriod=599497826)
  INFO root [2019-09-19 19:47:40,971] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:47:40,976] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:47:40,976] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:47:40,977] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:47:42,557] - 
  INFO root [2019-09-19 19:47:42,557] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:47:42,558] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:47:42,558] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:47:42,558] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:47:42,569] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61453 )
  INFO root [2019-09-19 19:47:42,569] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61453, SOMAXCONN=5.
  INFO root [2019-09-19 19:47:42,569] - server-configs:
  INFO root [2019-09-19 19:47:42,570] - 	internalTcpAddr : 192.168.202.54:61453
  INFO root [2019-09-19 19:47:42,570] - 	componentID : 5000
  WARN root [2019-09-19 19:47:42,570] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:47:44,076] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 18420 ----
  INFO root [2019-09-19 19:47:44,076] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:47:44,077] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:47:44,077] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:47:44,487] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:47:44,488] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:47:51,618] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:47:51,675] - Components::findComponents: found logger, addr:192.168.202.54:61455
  INFO root [2019-09-19 19:47:51,675] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:47:51,675] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:47:51,676] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:47:51,778] - Components::findComponents: found dbmgr, addr:192.168.202.54:61457
  INFO root [2019-09-19 19:47:51,779] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:47:51,779] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:47:51,779] - Components::findComponents: found dbmgr, addr:192.168.202.54:61457
  WARN root [2019-09-19 19:47:52,780] - Forwarding logs to logger[192.168.202.54:61455/0/0/0]...
 ERROR root [2019-09-19 19:54:45,049] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=855319333490, lastReceivedTime=855295565079, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:54:45,057] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:54:45,057] - ==> [2019-09-19 19:54:45 048] TCPPacketReceiver::processPendingEvents(192.168.202.54:61455): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:54:45,058] - ==> [2019-09-19 19:54:45 050] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=855319333490, lastReceivedTime=855295565079, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 19:54:45,131] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61457): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:54:45,131] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=855320148099, lastReceivedTime=855295325973, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 19:54:45,135] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:54:45,136] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:54:45,136] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:54:45,137] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:54:46,722] - 
  INFO root [2019-09-19 19:54:46,725] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:54:46,726] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:54:46,726] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:54:46,726] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:54:46,742] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61522 )
  INFO root [2019-09-19 19:54:46,742] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61522, SOMAXCONN=5.
  INFO root [2019-09-19 19:54:46,742] - server-configs:
  INFO root [2019-09-19 19:54:46,742] - 	internalTcpAddr : 192.168.202.54:61522
  INFO root [2019-09-19 19:54:46,742] - 	componentID : 5000
  WARN root [2019-09-19 19:54:46,745] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:54:48,250] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 8292 ----
  INFO root [2019-09-19 19:54:48,251] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:54:48,251] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:54:48,252] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:54:48,714] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:54:48,728] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 19:54:55,873] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 19:54:55,892] - Components::findComponents: found logger, addr:192.168.202.54:61521
  INFO root [2019-09-19 19:54:55,892] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:54:55,892] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:54:55,895] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 19:54:55,999] - Components::findComponents: found dbmgr, addr:192.168.202.54:61520
  INFO root [2019-09-19 19:54:55,999] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:54:55,999] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:54:55,999] - Components::findComponents: found dbmgr, addr:192.168.202.54:61520
  WARN root [2019-09-19 19:54:57,000] - Forwarding logs to logger[192.168.202.54:61521/0/0/0]...
 ERROR root [2019-09-19 19:58:33,489] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=857603722730, lastReceivedTime=857435952234, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 19:58:33,494] - The following logs sent to logger failed:
  WARN root [2019-09-19 19:58:33,495] - ==> [2019-09-19 19:58:33 485] TCPPacketReceiver::processPendingEvents(192.168.202.54:61521): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:58:33,495] - ==> [2019-09-19 19:58:33 489] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=857603722730, lastReceivedTime=857435952234, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 19:58:33,573] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61520): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 19:58:33,574] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=857604568692, lastReceivedTime=857436001404, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 19:58:33,576] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 19:58:33,577] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 19:58:33,578] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 19:58:33,578] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 19:58:35,165] - 
  INFO root [2019-09-19 19:58:35,165] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 19:58:35,166] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 19:58:35,166] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 19:58:35,166] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 19:58:35,176] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61599 )
  INFO root [2019-09-19 19:58:35,176] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61599, SOMAXCONN=5.
  INFO root [2019-09-19 19:58:35,176] - server-configs:
  INFO root [2019-09-19 19:58:35,176] - 	internalTcpAddr : 192.168.202.54:61599
  INFO root [2019-09-19 19:58:35,176] - 	componentID : 5000
  WARN root [2019-09-19 19:58:35,177] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 19:58:35,575] - Components::findLogger: found logger, addr:192.168.202.54:61597
  INFO root [2019-09-19 19:58:35,575] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 19:58:35,575] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 19:58:35,583] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 4324 ----
  INFO root [2019-09-19 19:58:35,584] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 19:58:35,585] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 19:58:35,585] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 19:58:36,019] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 19:58:36,021] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 19:58:42,129] - Forwarding logs to logger[192.168.202.54:61597/0/0/0]...
 ERROR root [2019-09-19 20:01:48,647] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=859555308691, lastReceivedTime=859420102562, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 20:01:48,661] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:01:48,661] - ==> [2019-09-19 20:01:48 637] TCPPacketReceiver::processPendingEvents(192.168.202.54:61597): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:01:48,661] - ==> [2019-09-19 20:01:48 648] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=859555308691, lastReceivedTime=859420102562, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 20:01:48,749] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61600): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:01:48,757] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=859556407081, lastReceivedTime=859420790871, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 20:01:48,763] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:01:48,767] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:01:48,767] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:01:48,768] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:01:50,822] - 
  INFO root [2019-09-19 20:01:50,847] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:01:50,849] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:01:50,849] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:01:50,849] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:01:50,943] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61665 )
  INFO root [2019-09-19 20:01:50,943] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61665, SOMAXCONN=5.
  INFO root [2019-09-19 20:01:50,943] - server-configs:
  INFO root [2019-09-19 20:01:50,943] - 	internalTcpAddr : 192.168.202.54:61665
  INFO root [2019-09-19 20:01:50,944] - 	componentID : 5000
  WARN root [2019-09-19 20:01:50,982] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 20:01:51,533] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20663, errorno=10048
  INFO root [2019-09-19 20:01:51,537] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 15304 ----
  INFO root [2019-09-19 20:01:51,537] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:01:51,537] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:01:51,538] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:01:52,021] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:01:52,021] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:01:59,179] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:01:59,280] - Components::findComponents: found logger, addr:192.168.202.54:61662
  INFO root [2019-09-19 20:01:59,280] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:01:59,280] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:01:59,281] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:01:59,383] - Components::findComponents: found dbmgr, addr:192.168.202.54:61668
  INFO root [2019-09-19 20:01:59,384] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:01:59,384] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:01:59,384] - Components::findComponents: found dbmgr, addr:192.168.202.54:61668
  WARN root [2019-09-19 20:02:00,385] - Forwarding logs to logger[192.168.202.54:61662/0/0/0]...
 ERROR root [2019-09-19 20:04:12,612] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=860994963834, lastReceivedTime=860778456307, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-19 20:04:12,615] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:04:12,616] - ==> [2019-09-19 20:04:12 610] TCPPacketReceiver::processPendingEvents(192.168.202.54:61662): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:04:12,617] - ==> [2019-09-19 20:04:12 614] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=860994963834, lastReceivedTime=860778456307, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-19 20:04:12,714] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61668): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:04:12,716] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=860996001907, lastReceivedTime=860778965506, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-19 20:04:12,724] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:04:12,728] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:04:12,730] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:04:12,731] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:05:38,181] - 
  INFO root [2019-09-19 20:05:38,181] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:05:38,182] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:05:38,182] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:05:38,183] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:05:38,193] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61734 )
  INFO root [2019-09-19 20:05:38,193] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61734, SOMAXCONN=5.
  INFO root [2019-09-19 20:05:38,193] - server-configs:
  INFO root [2019-09-19 20:05:38,193] - 	internalTcpAddr : 192.168.202.54:61734
  INFO root [2019-09-19 20:05:38,194] - 	componentID : 5000
  WARN root [2019-09-19 20:05:38,196] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:05:39,701] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 9840 ----
  INFO root [2019-09-19 20:05:39,702] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:05:39,702] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:05:39,702] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:05:40,168] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:05:40,169] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:05:47,343] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:05:47,371] - Components::findComponents: found logger, addr:192.168.202.54:61732
  INFO root [2019-09-19 20:05:47,372] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:05:47,372] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:05:47,373] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:05:47,477] - Components::findComponents: found dbmgr, addr:192.168.202.54:61733
  INFO root [2019-09-19 20:05:47,478] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:05:47,478] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:05:47,478] - Components::findComponents: found dbmgr, addr:192.168.202.54:61733
  WARN root [2019-09-19 20:05:48,479] - Forwarding logs to logger[192.168.202.54:61732/0/0/0]...
 ERROR root [2019-09-19 20:08:50,261] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=863771444067, lastReceivedTime=863648756526, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 20:08:50,263] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:08:50,264] - ==> [2019-09-19 20:08:50 258] TCPPacketReceiver::processPendingEvents(192.168.202.54:61732): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:08:50,264] - ==> [2019-09-19 20:08:50 262] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=863771444067, lastReceivedTime=863648756526, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 20:08:50,359] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61733): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:08:50,359] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=863772436494, lastReceivedTime=863650360330, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 20:08:50,362] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:08:50,363] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:08:50,363] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:08:50,363] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:22:58,205] - 
  INFO root [2019-09-19 20:22:58,208] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:22:58,209] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:22:58,209] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:22:58,209] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:22:58,228] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61853 )
  INFO root [2019-09-19 20:22:58,228] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61853, SOMAXCONN=5.
  INFO root [2019-09-19 20:22:58,228] - server-configs:
  INFO root [2019-09-19 20:22:58,228] - 	internalTcpAddr : 192.168.202.54:61853
  INFO root [2019-09-19 20:22:58,228] - 	componentID : 5000
  WARN root [2019-09-19 20:22:58,229] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:22:59,734] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 6692 ----
  INFO root [2019-09-19 20:22:59,735] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:22:59,735] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:22:59,735] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:23:00,196] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:23:00,197] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:23:07,337] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:23:07,392] - Components::findComponents: found logger, addr:192.168.202.54:61851
  INFO root [2019-09-19 20:23:07,392] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:23:07,392] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:23:07,394] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:23:07,497] - Components::findComponents: found dbmgr, addr:192.168.202.54:61855
  INFO root [2019-09-19 20:23:07,497] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:23:07,498] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:23:07,498] - Components::findComponents: found dbmgr, addr:192.168.202.54:61855
  WARN root [2019-09-19 20:23:08,498] - Forwarding logs to logger[192.168.202.54:61851/0/0/0]...
 ERROR root [2019-09-19 20:24:32,580] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=873194633362, lastReceivedTime=873150311571, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-19 20:24:32,583] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:24:32,583] - ==> [2019-09-19 20:24:32 578] TCPPacketReceiver::processPendingEvents(192.168.202.54:61851): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:24:32,583] - ==> [2019-09-19 20:24:32 580] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=873194633362, lastReceivedTime=873150311571, inactivityExceptionPeriod=599499745)
  WARN root [2019-09-19 20:24:32,672] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61855): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:24:32,678] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=873195617052, lastReceivedTime=873151620446, inactivityExceptionPeriod=599499745)
  INFO root [2019-09-19 20:24:32,694] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:24:32,701] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:24:32,701] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:24:32,702] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:25:22,529] - 
  INFO root [2019-09-19 20:25:22,530] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:25:22,531] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:25:22,531] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:25:22,531] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:25:22,541] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:61915 )
  INFO root [2019-09-19 20:25:22,542] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:61915, SOMAXCONN=5.
  INFO root [2019-09-19 20:25:22,542] - server-configs:
  INFO root [2019-09-19 20:25:22,542] - 	internalTcpAddr : 192.168.202.54:61915
  INFO root [2019-09-19 20:25:22,542] - 	componentID : 5000
  WARN root [2019-09-19 20:25:22,543] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:25:22,945] - Components::findLogger: found logger, addr:192.168.202.54:61913
  INFO root [2019-09-19 20:25:22,945] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:25:22,945] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:25:22,952] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 8308 ----
  INFO root [2019-09-19 20:25:22,957] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:25:22,958] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:25:22,958] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:25:23,486] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:25:23,497] - Components::process(): Request for the process of identity...
  WARN root [2019-09-19 20:25:29,626] - Forwarding logs to logger[192.168.202.54:61913/0/0/0]...
 ERROR root [2019-09-19 20:27:30,393] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=874972767770, lastReceivedTime=874893630453, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:27:30,395] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:27:30,395] - ==> [2019-09-19 20:27:30 388] TCPPacketReceiver::processPendingEvents(192.168.202.54:61913): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:27:30,395] - ==> [2019-09-19 20:27:30 394] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=874972767770, lastReceivedTime=874893630453, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 20:27:30,490] - TCPPacketReceiver::processPendingEvents(192.168.202.54:61914): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:27:30,502] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=874973851939, lastReceivedTime=874894028404, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:27:30,525] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:27:30,526] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:27:30,527] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:27:30,527] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:29:07,312] - 
  INFO root [2019-09-19 20:29:07,316] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:29:07,317] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:29:07,317] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:29:07,318] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:29:07,345] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62080 )
  INFO root [2019-09-19 20:29:07,345] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62080, SOMAXCONN=5.
  INFO root [2019-09-19 20:29:07,345] - server-configs:
  INFO root [2019-09-19 20:29:07,345] - 	internalTcpAddr : 192.168.202.54:62080
  INFO root [2019-09-19 20:29:07,346] - 	componentID : 5000
  WARN root [2019-09-19 20:29:07,362] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:29:08,867] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 9532 ----
  INFO root [2019-09-19 20:29:08,868] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:29:08,868] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:29:08,869] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:29:09,402] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:29:09,403] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:29:16,591] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:29:16,667] - Components::findComponents: found logger, addr:192.168.202.54:62085
  INFO root [2019-09-19 20:29:16,667] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:29:16,667] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:29:16,668] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:29:16,771] - Components::findComponents: found dbmgr, addr:192.168.202.54:62082
  INFO root [2019-09-19 20:29:16,771] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:29:16,771] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:29:16,777] - Components::findComponents: found dbmgr, addr:192.168.202.54:62082
  WARN root [2019-09-19 20:29:17,779] - Forwarding logs to logger[192.168.202.54:62085/0/0/0]...
 ERROR root [2019-09-19 20:30:42,963] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=876898468407, lastReceivedTime=876843127579, inactivityExceptionPeriod=599499297)
  INFO root [2019-09-19 20:30:42,965] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:30:42,966] - ==> [2019-09-19 20:30:42 963] TCPPacketReceiver::processPendingEvents(192.168.202.54:62085): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:30:42,966] - ==> [2019-09-19 20:30:42 964] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=876898468407, lastReceivedTime=876843127579, inactivityExceptionPeriod=599499297)
  WARN root [2019-09-19 20:30:43,055] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62082): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:30:43,060] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=876899433277, lastReceivedTime=876842836314, inactivityExceptionPeriod=599499297)
  INFO root [2019-09-19 20:30:43,085] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:30:43,088] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:30:43,088] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:30:43,088] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:30:44,660] - 
  INFO root [2019-09-19 20:30:44,663] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:30:44,664] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:30:44,664] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:30:44,664] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:30:44,691] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62159 )
  INFO root [2019-09-19 20:30:44,692] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62159, SOMAXCONN=5.
  INFO root [2019-09-19 20:30:44,692] - server-configs:
  INFO root [2019-09-19 20:30:44,692] - 	internalTcpAddr : 192.168.202.54:62159
  INFO root [2019-09-19 20:30:44,692] - 	componentID : 5000
  WARN root [2019-09-19 20:30:44,693] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 20:30:45,095] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20242, errorno=10048
  INFO root [2019-09-19 20:30:45,099] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 7452 ----
  INFO root [2019-09-19 20:30:45,099] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:30:45,100] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:30:45,100] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:30:45,535] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:30:45,535] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:30:52,681] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:30:52,727] - Components::findComponents: found logger, addr:192.168.202.54:62155
  INFO root [2019-09-19 20:30:52,727] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:30:52,727] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:30:52,728] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:30:52,830] - Components::findComponents: found dbmgr, addr:192.168.202.54:62156
  INFO root [2019-09-19 20:30:52,830] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:30:52,830] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:30:52,830] - Components::findComponents: found dbmgr, addr:192.168.202.54:62156
  WARN root [2019-09-19 20:30:53,831] - Forwarding logs to logger[192.168.202.54:62155/0/0/0]...
 ERROR root [2019-09-19 20:32:48,178] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=878150616054, lastReceivedTime=878115580182, inactivityExceptionPeriod=599500896)
  INFO root [2019-09-19 20:32:48,182] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:32:48,183] - ==> [2019-09-19 20:32:48 170] TCPPacketReceiver::processPendingEvents(192.168.202.54:62155): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:32:48,183] - ==> [2019-09-19 20:32:48 179] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=878150616054, lastReceivedTime=878115580182, inactivityExceptionPeriod=599500896)
  WARN root [2019-09-19 20:32:48,298] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62156): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:32:48,299] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=878151830133, lastReceivedTime=878115467923, inactivityExceptionPeriod=599500896)
  INFO root [2019-09-19 20:32:48,321] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:32:48,329] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:32:48,329] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:32:48,329] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:32:50,189] - 
  INFO root [2019-09-19 20:32:50,191] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:32:50,192] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:32:50,192] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:32:50,192] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:32:50,208] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62245 )
  INFO root [2019-09-19 20:32:50,208] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62245, SOMAXCONN=5.
  INFO root [2019-09-19 20:32:50,209] - server-configs:
  INFO root [2019-09-19 20:32:50,209] - 	internalTcpAddr : 192.168.202.54:62245
  INFO root [2019-09-19 20:32:50,209] - 	componentID : 5000
  WARN root [2019-09-19 20:32:50,209] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:32:51,717] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 16580 ----
  INFO root [2019-09-19 20:32:51,717] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:32:51,718] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:32:51,718] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:32:52,204] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:32:52,211] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:32:59,420] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:32:59,468] - Components::findComponents: found logger, addr:192.168.202.54:62244
  INFO root [2019-09-19 20:32:59,468] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:32:59,469] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:32:59,472] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:32:59,478] - Components::findComponents: found dbmgr, addr:192.168.202.54:62248
  INFO root [2019-09-19 20:32:59,478] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:32:59,478] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:32:59,579] - Components::findComponents: found dbmgr, addr:192.168.202.54:62248
  WARN root [2019-09-19 20:33:00,582] - Forwarding logs to logger[192.168.202.54:62244/0/0/0]...
 ERROR root [2019-09-19 20:34:50,133] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=879370172784, lastReceivedTime=879070781568, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 20:34:50,145] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:34:50,145] - ==> [2019-09-19 20:34:50 133] TCPPacketReceiver::processPendingEvents(192.168.202.54:62244): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:34:50,145] - ==> [2019-09-19 20:34:50 134] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=879370172784, lastReceivedTime=879070781568, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 20:34:50,237] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62248): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:34:50,239] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=879371230654, lastReceivedTime=879370659030, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 20:34:50,254] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:34:50,261] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:34:50,261] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:34:50,262] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:40:30,798] - 
  INFO root [2019-09-19 20:40:30,801] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:40:30,802] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:40:30,802] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:40:30,802] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:40:30,828] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62355 )
  INFO root [2019-09-19 20:40:30,828] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62355, SOMAXCONN=5.
  INFO root [2019-09-19 20:40:30,828] - server-configs:
  INFO root [2019-09-19 20:40:30,828] - 	internalTcpAddr : 192.168.202.54:62355
  INFO root [2019-09-19 20:40:30,828] - 	componentID : 5000
  WARN root [2019-09-19 20:40:30,829] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:40:32,333] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 16404 ----
  INFO root [2019-09-19 20:40:32,334] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:40:32,334] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:40:32,335] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:40:32,782] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:40:32,785] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:40:39,916] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:40:39,926] - Components::findComponents: found logger, addr:192.168.202.54:62352
  INFO root [2019-09-19 20:40:39,926] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:40:39,926] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:40:39,928] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:40:39,932] - Components::findComponents: found logger, addr:192.168.202.54:62352
  INFO root [2019-09-19 20:40:39,934] - Components::findComponents: found dbmgr, addr:192.168.202.54:62353
  INFO root [2019-09-19 20:40:39,935] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:40:39,935] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:40:40,036] - Components::findComponents: found dbmgr, addr:192.168.202.54:62353
  WARN root [2019-09-19 20:40:41,036] - Forwarding logs to logger[192.168.202.54:62352/0/0/0]...
 ERROR root [2019-09-19 20:44:42,383] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=885292664090, lastReceivedTime=885176663512, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:44:42,387] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:44:42,387] - ==> [2019-09-19 20:44:42 381] TCPPacketReceiver::processPendingEvents(192.168.202.54:62352): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:44:42,388] - ==> [2019-09-19 20:44:42 384] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=885292664090, lastReceivedTime=885176663512, inactivityExceptionPeriod=599499873)
  WARN root [2019-09-19 20:44:42,479] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62353): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:44:42,481] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=885293649358, lastReceivedTime=885176984447, inactivityExceptionPeriod=599499873)
  INFO root [2019-09-19 20:44:42,498] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:44:42,499] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:44:42,499] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:44:42,500] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 20:44:44,131] - 
  INFO root [2019-09-19 20:44:44,131] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:44:44,132] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:44:44,132] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:44:44,132] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:44:44,144] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62431 )
  INFO root [2019-09-19 20:44:44,145] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62431, SOMAXCONN=5.
  INFO root [2019-09-19 20:44:44,145] - server-configs:
  INFO root [2019-09-19 20:44:44,145] - 	internalTcpAddr : 192.168.202.54:62431
  INFO root [2019-09-19 20:44:44,145] - 	componentID : 5000
  WARN root [2019-09-19 20:44:44,145] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:44:45,651] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 1728 ----
  INFO root [2019-09-19 20:44:45,653] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:44:45,654] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:44:45,654] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:44:46,106] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:44:46,106] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:44:53,275] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:44:53,320] - Components::findComponents: found logger, addr:192.168.202.54:62434
  INFO root [2019-09-19 20:44:53,320] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:44:53,321] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:44:53,324] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:44:53,428] - Components::findComponents: found dbmgr, addr:192.168.202.54:62435
  INFO root [2019-09-19 20:44:53,429] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:44:53,429] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:44:53,429] - Components::findComponents: found dbmgr, addr:192.168.202.54:62435
  WARN root [2019-09-19 20:44:54,430] - Forwarding logs to logger[192.168.202.54:62434/0/0/0]...
  INFO root [2019-09-19 20:52:44,820] - EventDispatcher()::~EventDispatcher: Num timers = 2
  INFO root [2019-09-19 20:56:08,453] - 
  INFO root [2019-09-19 20:56:08,454] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 20:56:08,456] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 20:56:08,456] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 20:56:08,456] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 20:56:08,509] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62690 )
  INFO root [2019-09-19 20:56:08,509] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62690, SOMAXCONN=5.
  INFO root [2019-09-19 20:56:08,509] - server-configs:
  INFO root [2019-09-19 20:56:08,510] - 	internalTcpAddr : 192.168.202.54:62690
  INFO root [2019-09-19 20:56:08,510] - 	componentID : 5000
  WARN root [2019-09-19 20:56:08,515] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 20:56:10,023] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 1184 ----
  INFO root [2019-09-19 20:56:10,036] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 20:56:10,036] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 20:56:10,037] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 20:56:10,515] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 20:56:10,515] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 20:56:17,698] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 20:56:17,799] - Components::findComponents: found logger, addr:192.168.202.54:62686
  INFO root [2019-09-19 20:56:17,799] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:56:17,799] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:56:17,801] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 20:56:17,805] - Components::findComponents: found logger, addr:192.168.202.54:62686
  INFO root [2019-09-19 20:56:17,807] - Components::findComponents: found dbmgr, addr:192.168.202.54:62691
  INFO root [2019-09-19 20:56:17,807] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 20:56:17,808] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 20:56:17,907] - Components::findComponents: found dbmgr, addr:192.168.202.54:62691
  WARN root [2019-09-19 20:56:18,909] - Forwarding logs to logger[192.168.202.54:62686/0/0/0]...
 ERROR root [2019-09-19 20:58:35,548] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=893624316120, lastReceivedTime=893352844256, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:58:35,553] - The following logs sent to logger failed:
  WARN root [2019-09-19 20:58:35,554] - ==> [2019-09-19 20:58:35 545] TCPPacketReceiver::processPendingEvents(192.168.202.54:62686): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:58:35,554] - ==> [2019-09-19 20:58:35 549] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=893624316120, lastReceivedTime=893352844256, inactivityExceptionPeriod=599500000)
  WARN root [2019-09-19 20:58:35,660] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62691): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 20:58:35,666] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=893625493498, lastReceivedTime=893354441442, inactivityExceptionPeriod=599500000)
  INFO root [2019-09-19 20:58:35,689] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 20:58:35,692] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 20:58:35,692] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 20:58:35,693] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 21:07:18,192] - 
  INFO root [2019-09-19 21:07:18,192] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 21:07:18,193] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 21:07:18,193] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 21:07:18,194] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 21:07:18,204] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62775 )
  INFO root [2019-09-19 21:07:18,204] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62775, SOMAXCONN=5.
  INFO root [2019-09-19 21:07:18,204] - server-configs:
  INFO root [2019-09-19 21:07:18,205] - 	internalTcpAddr : 192.168.202.54:62775
  INFO root [2019-09-19 21:07:18,205] - 	componentID : 5000
  WARN root [2019-09-19 21:07:18,206] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 21:07:19,712] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 16776 ----
  INFO root [2019-09-19 21:07:19,713] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 21:07:19,713] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 21:07:19,713] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 21:07:20,124] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 21:07:20,124] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 21:07:27,255] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 21:07:27,332] - Components::findComponents: found logger, addr:192.168.202.54:62777
  INFO root [2019-09-19 21:07:27,333] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:07:27,333] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:07:27,334] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 21:07:27,338] - Components::findComponents: found dbmgr, addr:192.168.202.54:62780
  INFO root [2019-09-19 21:07:27,338] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:07:27,339] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:07:27,441] - Components::findComponents: found dbmgr, addr:192.168.202.54:62780
  WARN root [2019-09-19 21:07:28,442] - Forwarding logs to logger[192.168.202.54:62777/0/0/0]...
 ERROR root [2019-09-19 21:10:43,389] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=900902729789, lastReceivedTime=900650929592, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 21:10:43,397] - The following logs sent to logger failed:
  WARN root [2019-09-19 21:10:43,398] - ==> [2019-09-19 21:10:43 389] TCPPacketReceiver::processPendingEvents(192.168.202.54:62777): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:10:43,398] - ==> [2019-09-19 21:10:43 390] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=900902729789, lastReceivedTime=900650929592, inactivityExceptionPeriod=599500064)
  WARN root [2019-09-19 21:10:43,496] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62780): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:10:43,497] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=900903804218, lastReceivedTime=900652464595, inactivityExceptionPeriod=599500064)
  INFO root [2019-09-19 21:10:43,509] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 21:10:43,514] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 21:10:43,514] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 21:10:43,515] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 21:14:08,645] - 
  INFO root [2019-09-19 21:14:08,648] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 21:14:08,651] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 21:14:08,651] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 21:14:08,651] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 21:14:08,660] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:62897 )
  INFO root [2019-09-19 21:14:08,660] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:62897, SOMAXCONN=5.
  INFO root [2019-09-19 21:14:08,660] - server-configs:
  INFO root [2019-09-19 21:14:08,661] - 	internalTcpAddr : 192.168.202.54:62897
  INFO root [2019-09-19 21:14:08,661] - 	componentID : 5000
  WARN root [2019-09-19 21:14:08,664] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  INFO root [2019-09-19 21:14:10,172] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 2000 ----
  INFO root [2019-09-19 21:14:10,172] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 21:14:10,173] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 21:14:10,173] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 21:14:10,614] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 21:14:10,615] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 21:14:17,752] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 21:14:17,843] - Components::findComponents: found logger, addr:192.168.202.54:62898
  INFO root [2019-09-19 21:14:17,843] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:14:17,843] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:14:17,845] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 21:14:17,948] - Components::findComponents: found dbmgr, addr:192.168.202.54:62900
  INFO root [2019-09-19 21:14:17,948] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:14:17,949] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:14:17,955] - Components::findComponents: found dbmgr, addr:192.168.202.54:62900
  WARN root [2019-09-19 21:14:18,955] - Forwarding logs to logger[192.168.202.54:62898/0/0/0]...
 ERROR root [2019-09-19 21:38:36,824] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=917637076275, lastReceivedTime=917355531236, inactivityExceptionPeriod=599498913)
  INFO root [2019-09-19 21:38:36,827] - The following logs sent to logger failed:
  WARN root [2019-09-19 21:38:36,827] - ==> [2019-09-19 21:38:36 821] TCPPacketReceiver::processPendingEvents(192.168.202.54:62898): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:38:36,827] - ==> [2019-09-19 21:38:36 825] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=917637076275, lastReceivedTime=917355531236, inactivityExceptionPeriod=599498913)
  WARN root [2019-09-19 21:38:36,920] - TCPPacketReceiver::processPendingEvents(192.168.202.54:62900): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:38:36,923] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=917638064671, lastReceivedTime=917356362091, inactivityExceptionPeriod=599498913)
  INFO root [2019-09-19 21:38:36,930] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 21:38:36,932] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 21:38:36,934] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 21:38:36,935] - ThreadPool::destroy(): starting size 2.
  INFO root [2019-09-19 21:38:38,461] - 
  INFO root [2019-09-19 21:38:38,462] - -----------------------------------------------------------------------------------------


  INFO root [2019-09-19 21:38:38,463] - Resmgr::initialize: KBE_ROOT=D:/kbengine/
  INFO root [2019-09-19 21:38:38,463] - Resmgr::initialize: KBE_RES_PATH=D:/kbengine/kbe/res/;D:/kbengine/BattleOfBalls_assets/;D:/kbengine/BattleOfBalls_assets/scripts/;D:/kbengine/BattleOfBalls_assets/res/
  INFO root [2019-09-19 21:38:38,463] - Resmgr::initialize: KBE_BIN_PATH=D:/kbengine/kbe/bin/server/
  INFO root [2019-09-19 21:38:38,478] - NetworkInterface::initialize(INTERNAL-TCP): bound to all interfaces with default route interface on 192.168.202.54 ( 192.168.202.54:63158 )
  INFO root [2019-09-19 21:38:38,478] - NetworkInterface::initialize(INTERNAL-TCP): address 192.168.202.54:63158, SOMAXCONN=5.
  INFO root [2019-09-19 21:38:38,479] - server-configs:
  INFO root [2019-09-19 21:38:38,479] - 	internalTcpAddr : 192.168.202.54:63158
  INFO root [2019-09-19 21:38:38,479] - 	componentID : 5000
  WARN root [2019-09-19 21:38:38,482] - invalid UID(0) <= 0, please check UID for environment! automatically set to 41181.
  WARN root [2019-09-19 21:38:38,972] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20741, errorno=10048
  INFO root [2019-09-19 21:38:38,976] - ---- baseappmgr Version: 2.4.5. ScriptVersion: 0.1.0. Pythoncore: 3.7.3. Protocol: F070EF05A96B768F8BA788CE4BCBB097. Config: Debug X86. Built: 09:55:58 May  8 2019. AppID: 5000. UID: 41181. PID: 5728 ----
  INFO root [2019-09-19 21:38:38,976] - Server error_name: Intel(R) Core(TM) i5-6300HQ CPU @ 2.30GHz with 2097151kB RAM
  INFO root [2019-09-19 21:38:38,976] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2019-09-19 21:38:38,977] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
  INFO root [2019-09-19 21:38:39,436] - ---- baseappmgr is running ----
 DEBUG root [2019-09-19 21:38:39,439] - Components::process(): Request for the process of identity...
  INFO root [2019-09-19 21:38:46,599] - Components::findComponents: find logger(1)...
  INFO root [2019-09-19 21:38:46,608] - Components::findComponents: found logger, addr:192.168.202.54:63157
  INFO root [2019-09-19 21:38:46,608] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:38:46,608] - Components::addComponent[logger], uid=41181, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:38:46,609] - Components::findComponents: find dbmgr(2)...
  INFO root [2019-09-19 21:38:46,712] - Components::findComponents: found dbmgr, addr:192.168.202.54:63159
  INFO root [2019-09-19 21:38:46,712] - Components::addComponent: reset orderLog, uid=41181!
  INFO root [2019-09-19 21:38:46,712] - Components::addComponent[dbmgr], uid=41181, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2019-09-19 21:38:46,713] - Components::findComponents: found dbmgr, addr:192.168.202.54:63159
  WARN root [2019-09-19 21:38:47,713] - Forwarding logs to logger[192.168.202.54:63157/0/0/0]...
 ERROR root [2019-09-19 21:41:16,095] - Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=919229790856, lastReceivedTime=919152955709, inactivityExceptionPeriod=599500192)
  INFO root [2019-09-19 21:41:16,107] - The following logs sent to logger failed:
  WARN root [2019-09-19 21:41:16,107] - ==> [2019-09-19 21:41:16 095] TCPPacketReceiver::processPendingEvents(192.168.202.54:63157): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:41:16,107] - ==> [2019-09-19 21:41:16 096] Components::removeComponentByChannel: logger : 2000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=919229790856, lastReceivedTime=919152955709, inactivityExceptionPeriod=599500192)
  WARN root [2019-09-19 21:41:16,189] - TCPPacketReceiver::processPendingEvents(192.168.202.54:63159): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2019-09-19 21:41:16,192] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit(reason=TCPPacketReceiver::processRecv(): error=10054
)! Channel(timestamp=919230760608, lastReceivedTime=919153636139, inactivityExceptionPeriod=599500192)
  INFO root [2019-09-19 21:41:16,209] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
  INFO root [2019-09-19 21:41:16,213] - Shutdowner::onShutdown: shutting down
  INFO root [2019-09-19 21:41:16,213] - Shutdowner::onShutdownEnd: shutting down
 DEBUG root [2019-09-19 21:41:16,213] - ThreadPool::destroy(): starting size 2.
